usaon_benefit_tool/templates/base.html

```diff
@@ -52,11 +52,23 @@
   <footer>
     <hr />
     <div class="footer-items">
-      <p>&copy; USAON &amp; NSIDC</p>
+      <p>&copy; USAON &amp; NSIDC 2022</p>
```

Comment:
```suggestion
      <p>&copy; USAON &amp; NSIDC 2024</p>
```

Reply:
Actually, maybe the current year should also be a global variable so it automatically updates!

Reply:
Should I write the code for the updation of the date periodically?

Reply:
If you would like to, that'd be awesome! But I think it's cool to merge this PR if we just change the number to 2024. It feels like we should create a new ticket to auto-update the year. What do you think?

Reply:
Sure.